.github/workflows/lint.yml

```diff
@@ -11,14 +11,23 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Setup Python 3
+      - name: Setup Python 3.10
         uses: actions/setup-python@v4
         with:
           python-version: "3.10"
 
+      - name: Install Ruff
+        run: |
+          python -m pip install --upgrade pip
+          pip install ruff
+
       - name: Run black
         uses: psf/black@stable
 
+      - name: Lint with Ruff
+        run: |
+          ruff check . --format=github
+
       - name: Check for debugging print statements
         run: |
           if grep -rq "print(" mokapot; then
```

Comment:
this can also be done with ruff :P

Reply:
It is a rule not enabled by default (like 90% of ruff :P)

Enabling "T20" in the pyproject.toml should do
https://beta.ruff.rs/docs/rules/#flake8-print-t20

```
[tool.ruff]
# Enable pycodestyle (`E`) and Pyflakes (`F`) codes by default.
select = ["E", "F", "T20"]  # <<<<<<<<----- Here <<<<<<<<<<<<<<<<<<<<<<<<
ignore = []
```